receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,533 @@
+    type: string
+    enum:
+      - available
+      - used
+  host_effective:
+    value: effective
+    description: Whether the host is effective in the vCenter cluster
+    type: bool
+    enum:
+      - "true"
+      - "false"
+  latency_direction:
+    value: direction
+    description: The direction of disk latency.
+    enum:
+      - read
+      - write
+  latency_type:
+    value: type
+    description: The type of disk latency being reported.
+    enum:
+      - kernel
+      - device
+  throughput_direction:
+    value: direction
+    description: The direction of network throughput.
+    type: string
+    enum:
+      - transmitted
+      - received
+  vm_count_power_state:
+    value: power_state
+    description: Whether the virtual machines are powered on or off
+    type: string
+    enum:
+      - "on"
+      - "off"
+  vsan_latency_type:
+    value: type
+    description: The type of vSAN latency.
+    type: string
+    enum:
+      - read
+      - write
+  vsan_operation_type:
+    value: type
+    description: The type of vSAN operation.
+    type: string
+    enum:
+      - read
+      - write
+      - unmap
+  vsan_throughput_direction:
+    value: direction
+    description: The type of vSAN throughput.
+    type: string
+    enum:
+      - read
+      - write
+
+metrics:
+  vcenter.cluster.cpu.available:
+    enabled: true
+    description: The amount of CPU available to the cluster
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.cpu.effective:
+    enabled: true
+    description: The effective CPU available to the cluster. This value excludes memory from hosts in maintenance mode or are unresponsive.
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.cpu.used:
+    enabled: true
+    description: The amount of CPU used by the cluster
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.available:
+    enabled: true
+    description: The available memory of the cluster.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.effective:
+    enabled: true
+    description: The available memory of the cluster.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.used:
+    enabled: true
+    description: The memory that is currently used by the cluster
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.vsan.latency.avg:
+    enabled: true
+    description: Average latency of IOs generated by all vSAN clients in the cluster.
```

Comment:
Is this a gauge? If not, is there a timeframe associated with the value?

Reply:
The value returned from the current iteration of the code specifies startTime and endTime as the current time so this should be the "current value" but I suppose the aggregated value is not super interesting so perhaps a gauge fits better 👍